cmake/variables.cmake

```diff
@@ -139,6 +139,7 @@ list(APPEND ML_SYSTEM_INCLUDE_DIRECTORIES
   ${TORCH_INC}
   ${CMAKE_SOURCE_DIR}/3rd_party/include
   ${CMAKE_SOURCE_DIR}/3rd_party/eigen
+  ${CMAKE_SOURCE_DIR}/3rd_party/valijson/include
```

Comment:
Do we really need this? Is it just used for one test? Could we just make that test do something a bit different? Our integration tests with Java should be pretty good at confirming we really do output valid JSON. If we don't _have_ to include another 3rd party dependency then it's best not to.